app/src/shared/components/TextInput/index.jsx

```diff
@@ -19,6 +19,7 @@ const TextInput = ({ label, className, passwordStrengthUpdate, ...props }: Props
         {({ value, onFocusChange, setAutoCompleted, ...rest }) => (
             <TextField
                 {...rest}
+                className={className || ''}
```

Comment:
@mondoreale: Could you check this out? Your previous change just eats the className prop.

Reply:
Oh, did you have to re-add it? Why did I remove it, anyone knows? 😱

Reply:
Since you're here already, can you make it so that we don't have to default to an empty string? That's why I define types for `className` like so: `className?: ?string,`. 💃

Reply:
Hm, was it really there [before](https://github.com/streamr-dev/streamr-platform/commit/b18b9c3f24733f73397cb05b3a46011ab01d198c#diff-62165dda44e625e10de79e4117914a06R19)? Either way, let's add it somewhere. Otherwise it's being ignored all along.

Reply:
Oh, yeah, it's probably an older thing actually. Maybe we haven't used className with inputs so we missed it. Whatever, it's fixed now 😄